doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -176,7 +176,7 @@ DECT NR+
 Enhanced ShockBurst (ESB)
 -------------------------
 
-|no_changes_yet_note|
+* Added the Monitor mode feature.
```

Comment:
I would add a reference to 'esb_monitor_mode' from the ESB doc. But consider it just a suggestion.